server/src/test/java/io/crate/expression/scalar/GenRandomTextUUIDFunctionTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import io.crate.testing.MoreMatchers;
+
+public class GenRandomTextUUIDFunctionTest extends AbstractScalarFunctionsTest {
+
+    @Test
+    public void test_gen_random_text_uuid_returns_a_uuid() throws Exception {
+        assertEvaluate(
+            "gen_random_text_uuid()",
+            MoreMatchers.withFeature((String x) -> x.length(), "length", Matchers.is(20)));
+    }
```

Comment:
Could probably also use a fixed see to make the evaluation deterministic. WDYT?

Reply:
Yes I think that would be valuable.

Reply:
Oh, forgot about the time component in the uuid. Making it deterministic is not possible without also using some overriden `TimeBasedUUIDGenerator`. I'll leave it as it is.